TRLevelControlTests/Base/TestBase.cs

```diff
@@ -0,0 +1,243 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TRLevelControl;
+using TRLevelControl.Compression;
+using TRLevelControl.Model;
+
+namespace TRLevelControlTests;
+
+public class TestBase
+{
+    protected static readonly string _readPath = @"Levels\{0}\{1}";
+    protected static readonly string _writePath = @"Levels\{0}\{1}_TEMP{2}";
+
+    public static string GetReadPath(string level, TRGameVersion version)
+    {
+        return string.Format(_readPath, version.ToString(), level);
+    }
+
+    public static string GetWritePath(string level, TRGameVersion version)
+    {
+        return string.Format(_writePath, version.ToString(), Path.GetFileNameWithoutExtension(level), Path.GetExtension(level));
+    }
+
+    public static TRLevel GetTR1TestLevel()
+        => GetTR1Level("TEST1.PHD");
+
+    public static TRLevel GetTR1AltTestLevel()
+        => GetTR1Level("TEST2.PHD");
+
+    public static TR2Level GetTR2TestLevel()
+        => GetTR2Level("TEST1.TR2");
+
+    public static TR2Level GetTR2AltTestLevel()
+        => GetTR2Level("TEST2.TR2");
+
+    public static TR3Level GetTR3TestLevel()
+        => GetTR3Level("TEST1.TR2");
+
+    public static TR3Level GetTR3AltTestLevel()
+        => GetTR3Level("TEST2.TR2");
+
+    public static TR4Level GetTR4TestLevel()
+        => GetTR4Level("TEST1.TR4");
+
+    public static TR4Level GetTR4AltTestLevel()
+        => GetTR4Level("TEST2.TR4");
+
+    public static TR5Level GetTR5TestLevel()
+        => GetTR5Level("TEST1.TRC");
+
+    public static TR5Level GetTR5AltTestLevel()
+        => GetTR5Level("TEST2.TRC");
```

Comment:
These methods don't seem to be used anywhere?

Reply:
Ah yes, these will allow tests to run in GitHub actions, so we'll make custom levels to match up (we don't want to upload OG files for obvious reasons). Probably shouldn't have added these just yet, but they will be getting used soon, so we may as well keep them for now.